### PR TITLE
WCPay overview - display fees per payment method

### LINKED ISSUE #2377: Fees: update WCPay overview page to display fees per payment method
As we adding new payment methods having a different base and promo fees, we need to update how we render fees information on the overview page.

See also paJDYF-23t-p2#comment-6911 for more context.

### PR DESCRIPTION
**Pull request by dpaun1985**
Fixes #2377

#### Changes proposed in this Pull Request

This PR changes the way that fees are displayed on the WCPay overview page, under the "Account details" section. 
The fees were displayed ony for the card payment method, but since new payment methods have been added, having a different base and promo fees, we need to display only the discounts based on payment method.
So, the purpose is to replace the "base fee" (the standard card transaction fee) information with a new section "Active discounts" .

Overview page before changes:
Screenshot 2021-10-13 at 08 49 09

Overview page after changes:

Screenshot 2021-10-13 at 02 26 24

Design: 35wcSfexs3z8pxrj4daJnH-fi-0%3A1 

#### Testing instructions
- Add a discount fee with multiple payment methods. You can do this in 2 ways:
1. Select a fee structure on server side (wcpay/fees folder) and add it using the CLI command: 
`wp wcpay add_fee_structure_to_account <stripe_account_id> --fee_structure_id=<fee_structure_id>`
2. Add the fee manually in the database,`wcpay_test_account_fee_structures` table. In order to have the fee active `end_date` field should be in the future or null.

We should have only 1 fee structure active for an account. In theory, an account can have more than one account fee structure at the same time. However, the current implementation of promotions only allows a single fee structure to be applied at the point an account is created. So we're unlikely to see this in practice at the moment.

- Re-login or clear the account cache 
- Go to Payments -> Overview
- Check if the discount is displayed properly under the Account details->Active discounts  section for each payment method

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply)

### CODE REVIEW

**Comment by dpaun1985** [comment]
> @dpaun1985 I'm a bit unclear about how to test. A screenshot or two would help me navigate the changes.
> 
> For example:
> 
> > add more than 1 payment method
> 
> Where to do this?
> 
> > check if the fees are displayed properly for each payment method
> 
> Where on the Overview page to look for this?

It was a draft with the purpose to have an idea on how I started implementing the issue.
I just added more context.

File: client/components/account-status/index.js [hunk 1]
```diff
@@ -90,12 +106,12 @@ const AccountStatusDetails = ( props ) => {
 					depositsStatus={ accountStatus.depositsStatus }
 				/>
 			</AccountStatusItem>
-			<AccountStatusItem
-				align={ 'flex-start' }
-				label={ __( 'Base fee:', 'woocommerce-payments' ) }
-			>
-				<AccountFees accountFees={ accountFees } />
-			</AccountStatusItem>
+			{ 0 < discounts.length ? (
```
Comment (kalessil) [inline client/components/account-status/index.js:109]: Also `0 < discounts.length` -> `discounts.length` should work here.

File: client/components/account-status/account-fees/index.js [hunk 2]
```diff
@@ -34,24 +35,50 @@ const LearnMoreLink = ( { accountFees } ) => {
...
+};
+
+const AccountFees = ( props ) => {
+	const { accountFees } = props;
+
+	return (
+		<>
+			{ Object.entries( accountFees ).map( ( [ key, value ] ) => {
+				//ignore base and discount fields - still used for backwards compatibilityss
+				if (
```
Comment (kalessil) [inline client/components/account-status/account-fees/index.js:67]: `{}` are missing as well.

**Review by kalessil — CHANGES_REQUESTED** [review]
Look great overall! I left few comments and will do the final review+testing once those are addressed.

**Review by dmvrtx — APPROVED** [review]
Tested with account with no discount fees and then adding a discount fee and it worked as expected.

However couldn't test it with multiple discount fees: it looks like server doesn't return them properly.

**Comment by anu-rock** [comment]
LGTM :shipit: 

I've added a minor nitpicky comment.

**Review by kalessil — APPROVED** [review]
LGTM ! Also a suggestion from Anurag looks good.

**Comment by kalessil** [comment]
@dpaun1985, please add changelog entries before merging this PR. Thanks!

**Pull request merged**